discord/guild.py

```diff
@@ -4405,6 +4419,52 @@ async def edit_onboarding(
         new = await self._state.http.edit_onboarding(self.id, fields, reason=reason)
         return Onboarding(data=new, guild=self)
 
+    async def modify_incident_actions(
+        self,
+        *,
+        invites_disabled_until: datetime.datetime | None = MISSING,
+        dms_disabled_until: datetime.datetime | None = MISSING,
+        reason: str | None = MISSING,
+    ) -> IncidentsData:
+        """|coro|
+
+        Modify the guild's incident actions, controlling when invites or DMs
+        are re-enabled after being temporarily disabled. Requires
+        :attr:`~Permissions.manage_guild`.
```

Comment:
```Suggestion
        are re-enabled after being temporarily disabled. Requires
        the :attr:`~Permissions.manage_guild` permission.
```
Otherwise it'll show up as "Requires manage_guild` (iirc).